src/Microsoft.AspNetCore.Razor.Evolution/Legacy/CSharpCodeWriter.cs

```diff
@@ -632,5 +638,12 @@ public void Dispose()
                     .SetIndent(_startIndent);
             }
         }
+
+        private class NoopDisposable : IDisposable
```

Comment:
Suggest `NullDisposable` or `EmptyDisposable`. Feel free to ignore this comment.